invenio_communities/assets/semantic-ui/js/invenio_communities/curate.js

```diff
@@ -79,9 +79,13 @@ function ResultsGridItemTemplate({ result, index }) {
       <Card.Content>
         <Card.Header>{result.metadata.titles[0].title}</Card.Header>
         <Card.Description>
-          {_truncate(result.metadata.descriptions[0].description, {
-            length: 200,
-          })}
+          <div
+            dangerouslySetInnerHTML={{
+              __html: _truncate(result.metadata.descriptions[0].description, {
+                length: 200,
```

Comment:
how did you come up with this number ? why exactly 200?

Reply:
I didn't define the 200. I only added the dangerouslySetInnerHTML to ensure that the markup would be properly displayed. Should I use our truncate component and define a different criteria (like number of lines)?